.github/workflows/deploy-documentation.yml

```diff
@@ -15,28 +15,18 @@ jobs:
       AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
       AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
     steps:
-      - name: Check out code
+      - name: Checkout code
         uses: actions/checkout@v4
-        with:
-          fetch-depth: 2
 
-      - name: Setup Node.js environment
-        uses: actions/setup-node@v4
-        with:
-          node-version: 20
+      - name: Install NodeJS Dependencies
+        uses: ./.github/actions/install-node-dependencies
 
       - name: Setup rust
         uses: actions-rust-lang/setup-rust-toolchain@v1
 
       - name: Install tuono
         run: cargo install tuono@0.10.4
 
-      - name: Install pnpm
-        run: npm i -g pnpm
-
-      - name: Install dependencies
-        run: pnpm install
-
       - name: Build project
         working-directory: ./apps/documentation
         run: tuono build --static
```

Comment:
Another mystery... this part seems to hang now.

I also noticed now that in some workflow there is installed a fixed version of `tuono`.
Is this expected or the last available version should be used?

Reply:
Yeah I noticed that. I think the best would be a script that automatically infer the used tuono's version (by reading Cargo.toml) file. For the documentation I think we can move to the just deployed `0.12.3`

Reply:
I'm opening a PR now for updating the documentation version

Reply:
Merged. Could you try to rebase?

Reply:
All green!